src/Symfony/Bridge/Monolog/Handler/ConsoleHandler.php

```diff
@@ -122,7 +124,8 @@ public function onCommand(ConsoleCommandEvent $event)
     {
         $output = $event->getOutput();
         if ($output instanceof ConsoleOutputInterface) {
-            $output = $output->getErrorOutput();
+            $this->stdoutput = $output->getStdOutput();
```

Comment:
I don't think `getStdOutput` is a member of `Symfony\Component\Console\Output\ConsoleOutputInterface` or its parent `Symfony\Component\Console\Output\OutputInterface`